docs/configure/collections.md

```diff
@@ -0,0 +1,57 @@
+# :material-collage: Collections
+
+In addition to the systems displayed; EmulationStation (ES) has functionality to create custom collections that you can use to group and organize your game library in different ways.  For example; if you wanted to create a view that automatically grouped all of your Role Playing Games or one that only lists the games you are currently playing you can use ES collections to do that. 
+
+All of the options described below can be managed by pressing ++"START"++ anywhere in ES to open the Main Menu and then opening ++"Game Collection Settings"++
```

Comment:
I noticed that there's inconsistency throughout the wiki about how to emphasize menu options. I am **not** saying "my way or the highway", I just noticed it. Maybe we should decide on a consistent way and stick with it.

I think, the original wiki entries used `source code formatting` for menu options and ++"button formatting"++  for buttons. When I started writing, I used _italic formatting_ for menu options to distinguish them from `things you have to type` and ++"buttons you gotta press"++.  Therefore, I was the first one to break consistency, sorry for that!

However, I *really* do not have a preference here, I just think maybe it's time to make a decision about this, overhaul everything and maybe put it in `readme.md`, just to improve the overall impression people get while reading.